website/templates/profile/oauth_app_detail.mako

```diff
@@ -0,0 +1,110 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Application Detail</%def>
+<%def name="content()">
+<style type="text/css">
+    .border-box {
+        padding: 15px 10px;
+        border-radius: 4px;
+        border: solid #DDD;
+        border-width: 1px 0;
+        line-height: 1.1;
+        display: block;
+        margin-bottom: 1em;
+    }
+</style>
+
+<h2 class="page-header">OAuth Application Settings</h2>
+
+<div class="row">
+
+    <div class="col-sm-3">
+        <div class="panel panel-default">
+            <ul class="nav nav-stacked nav-pills">
```

Comment:
Hm, I see now that this is actually duplicated across all the profile settings sections.

If you're feeling ambitious, go ahead and refactor this out into a mako `include`. Otherwise, update this to be consistent with the new styling.